config/horizon-exporter.php

```diff
@@ -20,7 +20,9 @@
         \LKDevelopment\HorizonPrometheusExporter\Exporter\CurrentProccesesPerQueue::class,
         \LKDevelopment\HorizonPrometheusExporter\Exporter\FailedJobsPerHour::class,
         \LKDevelopment\HorizonPrometheusExporter\Exporter\HorizonStatus::class,
-        \LKDevelopment\HorizonPrometheusExporter\Exporter\RecentJobs::class
+        \LKDevelopment\HorizonPrometheusExporter\Exporter\RecentJobs::class,
+        \App\Metrics\FailedQsBatches::class,
```

Comment:
I chose to reuse the existing (badly named) Horizon Exporter here as there really is no justification to pull in another export library when we already have what we need (and it's just oddly named).